### PR TITLE
Clize applied to apply-lapse-rate CLI

### PR DESCRIPTION
**Pull request by MoseleyS**
Testing:
 - [x] Ran tests and they passed OK

### CODE REVIEW

File: lib/improver/cli/apply_lapse_rate.py [hunk 1]
```diff
@@ -85,13 +54,14 @@ def process(temperature, lapse_rate, source_orog, target_orog):
 
     Returns:
         iris.cube.Cube:
-            Cube after lapse rate has been applied to temperature data.
+            temperature Cube after lapse rate adjustment has been applied.
     """
+    from improver.argparser import ArgParser
```
Comment (benfitzpatrick) [inline lib/improver/cli/apply_lapse_rate.py]: Codacy Issue found: [Unused ArgParser imported from improver.argparser](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4665292)

**Comment by benfitzpatrick** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4665292&bid=15617305)

File: lib/improver/cli/apply_lapse_rate.py [hunk 2]
```diff
@@ -85,13 +54,14 @@ def process(temperature, lapse_rate, source_orog, target_orog):
...
+    from improver.lapse_rate import apply_gridded_lapse_rate
+    from improver.utilities.load import load_cube
```
Comment (benfitzpatrick) [inline lib/improver/cli/apply_lapse_rate.py]: Codacy Issue found: [Unused load_cube imported from improver.utilities.load](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4665292)

File: lib/improver/cli/apply_lapse_rate.py [hunk 3]
```diff
@@ -85,13 +54,14 @@ def process(temperature, lapse_rate, source_orog, target_orog):
...
+    from improver.utilities.save import save_netcdf
```
Comment (benfitzpatrick) [inline lib/improver/cli/apply_lapse_rate.py]: Codacy Issue found: [Unused save_netcdf imported from improver.utilities.save](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4665292)

File: lib/improver/cli/apply_lapse_rate.py [hunk 4]
```diff
@@ -85,13 +54,11 @@ def process(temperature, lapse_rate, source_orog, target_orog):
 
     Returns:
         iris.cube.Cube:
-            Cube after lapse rate has been applied to temperature data.
+            temperature Cube after lapse rate adjustment has been applied.
```
Comment (cgsandford) [inline lib/improver/cli/apply_lapse_rate.py]: `Temperature cube`?

File: lib/improver/tests/acceptance/test_apply_lapse_rate.py [hunk 5]
```diff
@@ -48,7 +48,7 @@ def test_basic(tmp_path):
             kgo_dir / "ukvx_lapse_rate.nc",
             kgo_dir / "ukvx_orography.nc",
             kgo_dir / "../highres_orog.nc",
-            output_path]
+            "--output", f"{output_path}"]
```
Comment (cgsandford) [inline lib/improver/tests/acceptance/test_apply_lapse_rate.py]: Do you need the `f"{output_path}"`?  I've just used `output_path` and it worked fine - does that not work in this case?

File: lib/improver/cli/apply_lapse_rate.py (hunk 4 above)
Comment (arh89) [reply lib/improver/cli/apply_lapse_rate.py]:
```suggestion
            Temperature cube after lapse rate adjustment has been applied.
```

File: lib/improver/tests/acceptance/test_apply_lapse_rate.py (hunk 5 above)
Comment (arh89) [reply lib/improver/tests/acceptance/test_apply_lapse_rate.py]:
```suggestion
            "--output", output_path]
```

File: lib/improver/tests/acceptance/test_apply_lapse_rate.py [hunk 6]
```diff
@@ -62,6 +62,6 @@ def test_realizations(tmp_path):
             kgo_dir / "enukx_lapse_rate.nc",
             kgo_dir / "enukx_orography.nc",
             kgo_dir / "../highres_orog.nc",
-            output_path]
+            "--output", f"{output_path}"]
```
Comment (arh89) [inline lib/improver/tests/acceptance/test_apply_lapse_rate.py]:
```suggestion
            "--output", output_path]
```

**Comment by arh89** [comment]
Looks good to me once @cgsandford is happy. (I've put her comments as suggestions to make it easier to commit the changes)

**Pull request merged**